From 541c17c975bc8a4f1f243e568006b0e26ae52c47 Mon Sep 17 00:00:00 2001
From: Sergey Fedorov <vital.had@gmail.com>
Date: Sun, 4 Feb 2024 09:28:26 +0800
Subject: [PATCH] darwin/Platform.c: use mach/port.h

Unbreak builds on macOS versions where _mach_port_t.h does not exist.
mach/port.h exists on every macOS and has needed defines.
---
 darwin/Platform.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git darwin/Platform.c darwin/Platform.c
index 387910e1..77b9bcdc 100644
--- darwin/Platform.c
+++ darwin/Platform.c
@@ -18,7 +18,7 @@ in the source distribution for its full text.
 #include <net/if_types.h>
 #include <net/route.h>
 #include <sys/socket.h>
-#include <sys/_types/_mach_port_t.h>
+#include <mach/port.h>
 
 #include <CoreFoundation/CFBase.h>
 #include <CoreFoundation/CFDictionary.h>
